libsolidity/interface/CompilerStack.cpp

```diff
@@ -1514,7 +1559,7 @@ bytes CompilerStack::createCBORMetadata(Contract const& _contract) const
 	else
 		solAssert(m_metadataHash == MetadataHash::None, "Invalid metadata hash");
 
-	if (experimentalMode)
+	if (experimentalMode || m_viaIR)
```

Comment:
Could also move the `m_viaIR` to the definition of the `experimentalMode` variable.

Reply:
I thought about that felt this is a different thing to "experimental features" which are pragma driven.

Reply:
Should I move it?